client/src/app/pages/identities/identities.tsx

```diff
@@ -173,35 +171,9 @@ export const Identities: React.FC = () => {
     },
   ];
 
-  const handleOnCancelUpdateIdentity = () => {
-    setRowToUpdate(undefined);
-  };
-
-  const editRow = (row: Identity) => {
-    setRowToUpdate(row);
-  };
-
-  const deleteRow = (row: Identity) => {
-    setIdentityIdToDelete(row.id);
-    setIsConfirmDialogOpen(true);
-  };
-
-  const handleOnIdentityCreated = (response: AxiosResponse<Identity>) => {
-    if (!identityToUpdate) {
-      pushNotification({
-        title: t("toastr.success.save", {
-          what: response.data.name,
-          type: t("terms.credential").toLowerCase(),
-        }),
-        variant: "success",
-      });
-    }
-
-    closeIdentityModal();
-  };
-
-  const handleOnIdentityUpdated = () => {
-    setRowToUpdate(undefined);
+  const closeCreateUpdateModal = () => {
+    setCreateUpdateModalState(null);
+    refetch;
```

Comment:
Is this supposed to be called?

Reply:
@ibolton336, normally we don't need it with `invalidateQueries()`
Let me double check...

Reply:
Cool - just saying if we do it needs perens 
```suggestion
    refetch();
```

Reply:
@ibolton336, redundant, I removed it.